src/modules/visitor-space/components/YearInput/YearInput.tsx

```diff
@@ -0,0 +1,27 @@
+import { TextInput, TextInputProps } from '@meemoo/react-components';
+import { FC } from 'react';
+
+import { Icon } from '@shared/components';
+
+import styles from './YearInput.module.scss';
+
+interface YearInputProps extends TextInputProps {
+	label?: string;
+}
+
+const YearInput: FC<YearInputProps> = (props) => (
+	<div className={styles['c-year-input']}>
+		<div className={styles['c-year-input__label']}>
+			<b>{props.label}</b>
+		</div>
+		<TextInput
+			style={{ textAlign: 'right' }}
+			iconStart={<Icon name="calendar" />}
+			placeholder="jjjj"
+			type="number"
+			{...props}
+		/>
+	</div>
```

Comment:
![image](https://user-images.githubusercontent.com/1710840/214013360-53d600bb-9b74-44bf-a17a-454a83cc9731.png)

Reply:
TODO: placeholder is missing

Reply:
placeholder is only visible when year is selected